src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderEmailService.java

```diff
@@ -601,10 +601,10 @@ private void sendEmailToSolicitorOrPostToRespondent(List<DynamicMultiselectListE
                 if (isSolicitorEmailExists(partyData)) {
                     try {
                         log.info("Trying to send email to {} via send grid service", partyData.getSolicitorEmail());
-                        sendgridService.sendEmailWithAttachments(
+                        sendgridService.sendEmailWithAttachments(YesOrNo.Yes,
```

Comment:
No ideal to add a new field for respondent solicitor, instead it should be added as part of email props